scipy/meson.build

```diff
@@ -184,6 +205,10 @@ elif lapack_name == 'openblas'
 else
   lapack = dependency(lapack_name)
 endif
+lapack_dep = declare_dependency(
+  dependencies: lapack,
+  compile_args: _args_blas
```

Comment:
If we're going to use this for both BLAS & LAPACK, let's call it `_args_blas_lapack`.